File: code/pete/javascript/solutions/pick3-blackjack/blackjack.py
card_dict = {
    'A': 1,
    '2': 2,
    '3': 3,
    '4': 4,
    '5': 5,
    '6': 6,
    '7': 7,
    '8': 8,
    '9': 9,
    '10': 10,
    'J': 10,
    'Q': 10,
    'K': 10,
}

card1 = input('Enter a card: ')
card2 = input('Enter another card: ')
card3 = input('Enter yet another card: ')

cards = [card1, card2, card3]

total = 0

for card in cards:
    total += card_dict[card]

print('your total is: ' + str(total))

if total < 17:
    print('hit')
elif total < 21:
    print('stay')
elif total == 21:
    print('blackjack')
else:
    print('bust')

File: code/dbrunken/JavaScript/Labs/lab1/blackjack/part 1/blackjack.js
// give advice on blackjack

let deck = {
    'A' : 1,
    2 : 2,
    3 : 3,
    4 : 4,
    5 : 5,
    6 : 6,
    7 : 7,
    8 : 8,
    9 : 9,
    10 : 10,
    'J' : 10,
    'Q' : 10,
    'K' : 10,
}

const card1 = prompt('enter first card ')
const card2 = prompt('enter second card ')
const card3 = prompt('enter third card ')

const hand = [card1, card2, card3]

let total = 0

for (let i=0; i<3; ++i) {
    total += deck[hand[i]]
}

alert('your total is: ' + total)

if (total < 17) {
    alert('hit')
} else if (total < 21) {
    alert('stay')
} else if (total === 21) {
    alert('blackjack')
} else {
    alert('bust')
}